From cbc482efe25a4dfb20d2ec7d92e27ca026dc3a9b Mon Sep 17 00:00:00 2001
From: hienhuynh <hien.huynh.px@rvc.renenas.com>
Date: Thu, 4 Oct 2018 16:54:01 +0700
Subject: [PATCH 103/445] ARM: dts: r8a7742-iwg21d-q7: Add gpio properties for
 detecting sound jack

Add hp-det-gpios to detect headphone jack plugging state.
Add mic-det-gpios to detect mic jack plugging state.

Signed-off-by: hienhuynh <hien.huynh.px@rvc.renenas.com>
---
 arch/arm/boot/dts/r8a7742-iwg21d-q7.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7742-iwg21d-q7.dts b/arch/arm/boot/dts/r8a7742-iwg21d-q7.dts
index 50b225a..3003063 100644
--- a/arch/arm/boot/dts/r8a7742-iwg21d-q7.dts
+++ b/arch/arm/boot/dts/r8a7742-iwg21d-q7.dts
@@ -127,6 +127,8 @@
 		VDDA-supply = <&reg_3p3v>;
 		VDDIO-supply = <&reg_3p3v>;
 		VDDD-supply = <&reg_1p5v>;
+		hp-det-gpios = <&gpio4 19 GPIO_ACTIVE_LOW>;
+		mic-det-gpios = <&gpio0 26 GPIO_ACTIVE_LOW>;
 	};
 };
 
-- 
2.7.4

